lib/hooks/api/useContributorPullRequests.ts

```diff
@@ -15,7 +15,8 @@ export function getContributorPRUrl(
   topic: string,
   repoIds: number[] = [],
   limit = 8,
-  range = 30
+  range = 30,
+  mostRecent = false
```

Comment:
Refeactor (non-blocking): There's quite a few parameters now. Consider promoting them to one option parameter with all the current parameters.